added more detailed documentation

adamseth2 opened the pull request:
Added more in-depth documentation.

Some of the read me mentioned stuff that is not yet implemented, like switching web socket IP address, but should be implemented within the next two weeks.

uellenberg commented on README.md:
```diff
@@ -1,55 +1,45 @@
-# Mission Control
+# **Mission Control**
```
Don't bold headers; they already have a distinct style to them.

adamseth2 replied:
This was a weird thing done on the other documentation for some reason, so I copied it. Personally yeah kind of redundant. Unbolded all headers

uellenberg commented on README.md:
```diff
@@ -1,55 +1,45 @@
 
-## Recommended IDE Setup
+- **Testing Locally (on personal Docker):**
+  - Visit [Mission Control](https://www.trickfirerobotics.org/mission-control/) and follow the URC instructions to run the Docker container. Click on the Navbar websocket icon to switch to local development
```
I know you acknowledged this, but I don't think we should give incorrect information on our documentation. It will be implemented in the future, but if we tell people to do it now, then it'll get confusing.

adamseth2 replied:
Looks like the club is switching to Notion for documentation. Will talk at the software meeting about where software documentation for non-software people should go. For now, deleted that part and saved it in a sticky note for later.

uellenberg commented on README.md:
```diff
@@ -1,55 +1,45 @@
-2. Run `git clone https://github.com/TrickfireRobotics/mission-control.git` in terminal **(not in the URC Docker Container)**
-3. Run `cd mission-control` to open the repository
-4. Run `npm install` to install dependencies
+Setup router and ensure connect to trickfirerouter on your computer. The router password is on the router itself Turn the rover computer on by plugging in the c port charger. You will know the computer is on if the LED lights up.
```
Grammar.

adamseth2 replied:
Fixed all of the typos and grammar stuff. Moved this particular one to TrickfireRobotics/urc-2023#81

uellenberg commented on README.md:
```diff
@@ -1,55 +1,45 @@
-3. Run `cd mission-control` to open the repository
-4. Run `npm install` to install dependencies
+Setup router and ensure connect to trickfirerouter on your computer. The router password is on the router itself Turn the rover computer on by plugging in the c port charger. You will know the computer is on if the LED lights up.
+Run the commands below in Powershell or in your OS terminal. When prompted for a password, enter `trickfire`
```
I know it isn't exactly the most secure password, but should we really be putting it in our public GitHub? I guess it's already here now.

uellenberg commented on README.md:
````diff
@@ -1,55 +1,45 @@
-   - Run `./build.sh` and `./launch.sh` commands in the container terminal
-3. If the Websocket circle becomes **green**, you are connected!
+```
+Open Powershell or Terminal
+> ssh trickfire@192.168.0.145
+In Bash Now
+> cd ~/urc-2023
+> sudo ./container_launch.sh
+In Docker Now
+> cd /home/trickfire/urc-2023
+> ./build.sh
+> ./launch.sh
````
This feels like something that should be included in the urc-2023 repo, or as internal documentation. It's specific to the configuration on the rover, and doesn't really have that much to do with mission-control.

adamseth2 replied:
Fixed all of the typos and grammar stuff. Moved this particular one to TrickfireRobotics/urc-2023#81

uellenberg reviewed (changes requested):
Looks good, thank you for working on all this documentation! Just a few things I noticed while going through it.

adamseth2 commented:
For the issue caused by my tiredness, I rebased and changed the line. It will always be there, but it is the easiest way without having to do more complex stuff.
